src/test/munit/batch-read-write-test-suite.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns="http://www.mulesoft.org/schema/mule/core"
+      xmlns:batch="http://www.mulesoft.org/schema/mule/batch"
+      xmlns:munit="http://www.mulesoft.org/schema/mule/munit"
+      xmlns:munit-tools="http://www.mulesoft.org/schema/mule/munit-tools"
+      xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+      xmlns:marklogic="http://www.mulesoft.org/schema/mule/marklogic"
+	xsi:schemaLocation="
+		http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+		http://www.mulesoft.org/schema/mule/munit http://www.mulesoft.org/schema/mule/munit/current/mule-munit.xsd
+		http://www.mulesoft.org/schema/mule/munit-tools  http://www.mulesoft.org/schema/mule/munit-tools/current/mule-munit-tools.xsd
+    http://www.mulesoft.org/schema/mule/batch http://www.mulesoft.org/schema/mule/batch/current/mule-batch.xsd
+		http://www.mulesoft.org/schema/mule/marklogic http://www.mulesoft.org/schema/mule/marklogic/current/mule-marklogic.xsd
+">
+
+  <!--  TODO - We probably need to read this information from a single file.-->
+  <marklogic:config name="MarkLogicConfig" configId="MarkLogicConfig">
+    <marklogic:connection
+        host="localhost"
+        port="8022"
+        username="mule2-test-user"
+        password="password"
+        authenticationType="DIGEST"/>
+  </marklogic:config>
+
+  <flow name="readTenDocsWriteTwoBatches" doc:id="63d4c67d-b038-4455-a4c1-f862de4bb735">
+    <marklogic:search-documents config-ref="MarkLogicConfig" collection="batch-input"/>
+    <batch:job jobName="BatchJob">
+      <batch:process-records>
+        <batch:step name="Step1">
+          <batch:aggregator size="5">
+            <marklogic:write-batch config-ref="MarkLogicConfig"/>
+          </batch:aggregator>
+        </batch:step>
+      </batch:process-records>
+    </batch:job>
+  </flow>
+
+	<munit:config name="batch-read-write-test-suite" />
```

Comment:
Does this do anything, or is it just required to be here?

Reply:
This is required. I don't know what it does.